src/lib/docs/get-docs-nav-items.ts

```diff
@@ -5,6 +5,18 @@ import {
 	RootDocsPath,
 } from 'types/products'
 
+/**
+ * Check if docs has more than one sub-item
+ * to determine whether or not to show dropdown in nav
+ */
+
+export function checkDocsNavHasItems(
+	currentProduct: Pick<ProductData, 'rootDocsPaths' | 'docsNavItems'>
+): boolean {
+	const { docsNavItems, rootDocsPaths } = currentProduct
+	return !(!docsNavItems && rootDocsPaths.length === 1)
```

Comment:
```suggestion
	return Boolean(docsNavItems) && rootDocsPaths.length > 1
```
💭 small thought: to make a logic a bit easier to read